jujugui/static/gui/src/app/components/model-actions/model-actions.js

```diff
@@ -127,8 +135,10 @@ class ModelActions extends React.Component {
 
 ModelActions.propTypes = {
   acl: PropTypes.object.isRequired,
+  address: PropTypes.string,
```

Comment:
can you combine these two props under an object - maybe even a shapeup object since they are both for the terminal, that way we can pass new data into it without having to modify every component between the app and the terminal